scrapy/selector/__init__.py

```diff
@@ -1,4 +1,6 @@
 """
 Selectors
 """
-from scrapy.selector.unified import *  # noqa: F401
+
+# top-level imports
+from scrapy.selector.unified import Selector, SelectorList
```

Comment:
This is safe, these two classes are the only ones that have been exposed from [the very beginning](https://github.com/scrapy/scrapy/blame/2.1.0/scrapy/selector/unified.py#L11).